deployment/terraform/dataspace/main.tf

```diff
@@ -76,17 +79,20 @@ resource "azurerm_container_group" "registration-service" {
     memory = var.container_memory
 
     ports {
-      port     = local.edc_default_port
+      port     = local.registration_service_port
       protocol = "TCP"
     }
 
     environment_variables = {
-      EDC_CONNECTOR_NAME = local.connector_name
+      EDC_CONNECTOR_NAME      = local.connector_name
+      JWT_AUDIENCE            = "http://${local.registration_service_dns_label}.${var.location}.azurecontainer.io:${local.registration_service_port}${local.registration_service_url_prefix}"
```

Comment:
nit: maybe we can also have a local var as registration_service_api_url and use as 

JWT_AUDIENCE = local.registration_service_api_url